### PR TITLE
Pcm header service

### PR DESCRIPTION
**Pull request by vrathee**
@renujhamtani @gandhikeyur Please Review

### CODE REVIEW

**Comment by gandhikeyur** [comment]
Reviewed

File: app/common/controllers/header.js [hunk 1]
```diff
@@ -27,7 +25,6 @@ angular.module('RedhatAccess.header').controller('HeaderController', [
         $scope.init = function () {
             HeaderService.pageLoadFailure = false;
             HeaderService.showPartnerEscalationError = false;
-            CaseService.sfdcIsHealthy = COMMON_CONFIG.sfdcIsHealthy;
```
Comment (aritrod) [inline app/common/controllers/header.js:30]: @vrathee everything looks fine, just ensure CaseService.sfdcIsHealthy is not being used any where outside common module. I doubt it but please check. If it is , replace with COMMON_CONFIG.sfdcIsHealthy.

**Comment by vrathee** [comment]
@ardasgup Have made changes to PCM where we use it.. will send that PR for review shortly :)

**Comment by vrathee** [comment]
@ardasgup Have made changes to PCM where we use it.. will send that PR for review shortly :)

**Pull request merged**